prometheus/prometheus.yml

```diff
@@ -0,0 +1,24 @@
+global:
```

Comment:
this is an overall configuration file for current prometheus configuration ? Add comment or description of the file and what it is for in comment/Readme.

Reply:
added a comment